src/content/ja/ai-capabilities/controlnet.md

```
@@ -0,0 +1,76 @@
+
+---
+
+## 代表的なControlNetの種類
+
+ControlNet で扱える「追加情報」はアイデア次第でいくらでも増やせますが、よく使われるものはある程度パターンがあります。
+代表的なものだけ挙げておきます。
+
+### openpose（ポーズ / 棒人間）
+棒人間やスケルトンで、人やキャラのポーズを指定します。
+
+![](https://gyazo.com/637abbf2514e4c973b519053ae5809cd){gyazo=image} ![](https://gyazo.com/aa98af3564647910d9c8b647a9ecbd16){gyazo=image}
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)